config/oauth/deployment.yaml

```diff
@@ -65,6 +65,7 @@ spec:
             - "--upstream=http://127.0.0.1:8080/"
             - "--logtostderr=true"
             - "--v=10"
+            - "--http2-disable=true"
```

Comment:
Doesn't seem to match with CLI arg name

Reply:
same as above, this is for kube-rbac-proxy where the arg is called this name.